packages/psss/src/components/Header.js

```diff
@@ -129,14 +142,19 @@ export const Header = ({ Title, back, ExportModal }) => {
             )}
             {Title}
           </div>
-          {ExportModal && (
-            <>
-              <LightOutlinedButton onClick={() => setIsModalOpen(true)} startIcon={<SaveAlt />}>
-                Export Data
-              </LightOutlinedButton>
-              <ExportModal isOpen={isModalOpen} handleClose={() => setIsModalOpen(false)} />
-            </>
-          )}
+          <div>
+            <TupaiaButton href="https://tupaia.org" startIcon={<TupaiaIcon />}>
+              Preview Tupaia
```

Comment:
Not sure about the wording "Preview". @enunan? It's more like "Visit" or "See" or "View"

Reply:
Also another good question that @Klaus-S rasied. Should we open Tupaia.org in a new tab or just take them to it in the existing tab? It's Easy enough to implement either way.

Reply:
New tab IMO